exercises/simple-linked-list/Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+name = "simple_linked_list"
+version = "1.0.0"
```

Comment:
since this is not based on any problem-specifications 1.0.0 (https://github.com/exercism/problem-specifications/tree/master/exercises/simple-linked-list not existing), this must be 0.0.0 or 0.1.0 (your choice), so that when 1.0.0 is created we know that we need to check whether we are in compliance with it.

If we do not do this, we could get out of date with problem-specifications and not know about it.